src/js/icons/Instagram.js

```diff
@@ -4,6 +4,6 @@ import { StyledIcon } from '../StyledIcon';
,23.0633712 21.3187732,22.6530513 21.985936,21.985936 C22.6530513,21.3187732 23.0633712,20.6495621 23.3700394,19.860456 C23.6666567,19.0971676 23.8695303,18.2248447 23.9277867,16.9475847 C23.9861861,15.6676571 24,15.2590043 24,12.0000238 C24,8.74099567 23.9861861,8.33234294 23.9277867,7.05241535 C23.8695303,5.77515526 23.6666567,4.90283244 23.3700394,4.13954402 C23.0633712,3.35043794 22.6530513,2.68122682 21.985936,2.01406396 C21.3187732,1.34694874 20.6495621,0.936628817 19.860456,0.630008276 C19.0971676,0.333343257 18.2248447,0.130469732 16.9475847,0.0722132578 C15.6676571,0.0138138818 15.2590043,0 12.0000238,0 Z M12.0000238,2.1621583 C15.2041299,2.1621583 15.5836782,2.17440026 16.8490298,2.23213276 C18.0190179,2.28548292 18.6544088,2.48097316 19.0772565,2.64531072 C19.6373856,2.86299845 20.0371308,3.12303286 20.4570251,3.54297487 C20.8769671,3.96286924 21.1370016,4.3626144 21.3546893,4.92274348 C21.5190268,5.34559117 21.7145171,5.9809821 21.7678672,7.15097025 C21.8255997,8.41632182 21.8378417,8.79587013 21.8378417,12.0000238 C21.8378417,15.2041299 21.8255997,15.5836782 21.7678672,16.8490298 C21.7145171,18.0190179 21.5190268,18.6544088 21.3546893,19.0772565 C21.1370016,19.6373856 20.8769671,20.0371308 20.4570251,20.4570251 C20.0371308,20.8769671 19.6373856,21.1370016 19.0772565,21.3546893 C18.6544088,21.5190268 18.0190179,21.7145171 16.8490298,21.7678672 C15.5838687,21.8255997 15.204368,21.8378417 12.0000238,21.8378417 C8.79563196,21.8378417 8.41617891,21.8255997 7.15097025,21.7678672 C5.9809821,21.7145171 5.34559117,21.5190268 4.92274348,21.3546893 C4.3626144,21.1370016 3.96286924,20.8769671 3.54297487,20.4570251 C3.1230805,20.0371308 2.86299845,19.6373856 2.64531072,19.0772565 C2.48097316,18.6544088 2.28548292,18.0190179 2.23213276,16.8490298 C2.17440026,15.5836782 2.1621583,15.2041299 2.1621583,12.0000238 C2.1621583,8.79587013 2.17440026,8.41632182 2.23213276,7.15097025 C2.28548292,5.9809821 2.48097316,5.34559117 2.64531072,4.92274348 C2.86299845,4.3626144 3.12303286,3.96286924 3.54297487,3.54297487 C3.96286924,3.12303286 4.3626144,2.86299845 4.92274348,2.64531072 C5.34559117,2.48097316 5.9809821,2.28548292 7.15097025,2.23213276 C8.41632182,2.17440026 8.79587013,2.1621583 12.0000238,2.1621583 Z M12.0000238,16 C9.79085069,16 8,14.2091493 8,12.0000238 C8,9.79085069 9.79085069,8 12.0000238,8 C14.2091493,8 16,9.79085069 16,12.0000238 C16,14.2091493 14.2091493,16 12.0000238,16 Z M12.0000238,5.8378417 C8.59671206,5.8378417 5.8378417,8.59671206 5.8378417,12.0000238 C5.8378417,15.4032879 8.59671206,18.1621583 12.0000238,18.1621583 C15.4032879,18.1621583 18.1621583,15.4032879 18.1621583,12.0000238 C18.1621583,8.59671206 15.4032879,5.8378417 12.0000238,5.8378417 Z M19.8456418,5.59438394 C19.8456418,6.38968246 19.2009146,7.03436203 18.4056161,7.03436203 C17.6103652,7.03436203 16.965638,6.38968246 16.965638,5.59438394 C16.965638,4.79908543 17.6103652,4.15435822 18.4056161,4.15435822 C19.2009146,4.15435822 19.8456418,4.79908543 19.8456418,5.59438394 Z' />
```

Comment:
Why is the Instagram fill a url?

Reply:
Check out my comment above

Reply:
I don't think that will work. Take a look at instagram.svg. "instagram-a" is an internal reference to a linear gradient that is being stripped when `generate-icons` is run.

Reply:
well, I agree it looks strange, but if you do a fresh clone run `yarn install && yarn generate-icons` this file will be updated. doesn't mean we need to change something on the svg?

Reply:
I removed the Instagram changes to eliminate the bottleneck of this PR. we'll discuss it offline. @ericsoderberghp